modules/ocm_cluster_rosa_classic/operator_roles/cloud_credential_role.tf

```diff
@@ -0,0 +1,32 @@
+resource "aws_iam_role" "cloud-credential_role" {
```

Comment:
By having the operator roles trust statically setup in these tf files will require us to update this everytime we need a new role, correct?

Could there be some way to generate these dynamically? Also depending on the version chosen for the cluster there are different operator roles not sure if I got how that is being handled

Reply:
@gdbranco for the new role we will need to add one more `tf` file that includes the new IAM role resource.
According to your last question - only the policies can be changed between cluster version (as I understood). when I asked about it in the sync meeting the answer was that roles cannot be changed